src/main/java/org/openrewrite/java/liberty/ServerName.java

```diff
@@ -36,16 +36,24 @@ public String getDescription() {
     }
 
     private static final String SERVER_NAME = "com.ibm.websphere.runtime.ServerName";
-    private static final MethodMatcher getDisplayName = new MethodMatcher(SERVER_NAME + " getDisplayName()");
-    private static final MethodMatcher getFullName = new MethodMatcher(SERVER_NAME + " getFullName()");
+    private static final MethodMatcher GET_DISPLAY_NAME = new MethodMatcher(SERVER_NAME + " getDisplayName()");
+    private static final MethodMatcher GET_FULL_NAME = new MethodMatcher(SERVER_NAME + " getFullName()");
+
+    private static final String ADMIN_SERVICE = "com.ibm.websphere.management.AdminService";
+    private static final MethodMatcher GET_PROCESS_NAME = new MethodMatcher(ADMIN_SERVICE + " getProcessName()");
+
+    private static final String RAS_HELPER = "com.ibm.ejs.ras.RasHelper";
+    private static final MethodMatcher GET_SERVER_NAME = new MethodMatcher(RAS_HELPER + " getServerName()");
 
     @Override
     public TreeVisitor<?, ExecutionContext> getVisitor() {
         return new JavaVisitor<ExecutionContext>() {
```

Comment:
This can be an isomorphic visitor, since we are only returning LST nodes with the same type.